presto-main/src/test/java/io/prestosql/sql/query/QueryAssertions.java

```diff
@@ -68,9 +69,10 @@ public void assertFails(@Language("SQL") String sql, @Language("RegExp") String
             fail(format("Expected query to fail: %s", sql));
         }
         catch (RuntimeException exception) {
-            if (!nullToEmpty(exception.getMessage()).matches(expectedMessageRegExp)) {
-                fail(format("Expected exception message '%s' to match '%s' for query: %s", exception.getMessage(), expectedMessageRegExp, sql), exception);
-            }
+            exception.addSuppressed(new Exception("Query: " + sql));
```

Comment:
Is this needed? Is this going to be printed by `assertj`? Maybe please add a comment.

Reply:
yes, will be printed 
and yes it is needed (those SQLs are often very important)

it's intentional, so what else should i contain the comment?

Reply:
Ok. Thanks